cdk/lib/frontend-stack.ts

```diff
@@ -107,11 +107,11 @@ export class FrontendStack extends cdk.Stack {
      */
     const frontendDeploy = new s3Deploy.BucketDeployment(
       this,
-      "DeployFrontend",
+      "Deploy-Frontend",
```

Comment:
Is this just a simple name change?

Reply:
@ajmokotoff Yes, because of this [issue](https://github.com/aws/aws-cdk/issues/7128).  You have to change the resource name when you change the memory size